docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>About the third point: I would say Y | mu, theta ~ D(mu, theta) or something like that. "mu" is the mean parameter and "theta" is (possibly) a vector that contains all the other "nuisance" parameters. </p><p> </p><p>As it is written now the second and third points both talk about the mean.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yes, good point. I used the following notation: Y ~ D(mu, theta). </p>